docs/pt/docs/external-links.md

```diff
@@ -0,0 +1,123 @@
+# Links externos e Artigos
+
+**FastAPI** tem uma grande comunidade em crescimento constante.
+
+Existem muitas postagens, artigos, ferramentas e projetos relacionados ao **FastAPI**.
+
+Aqui tem uma lista, incompleta, de algumas delas.
+
+!!! tip
+    Se você tem um artigo, projeto, ferramenta ou qualquer coisa relacionada ao **FastAPI** que ainda não está listada aqui, crie um <a href="https://github.com/tiangolo/fastapi/edit/master/docs/external-links.md" class="external-link" target="_blank">_Pull Request_ adicionando ele</a>.
```

Comment:
I'm fine with the first current `adicionando ele`, but if you think the second one is best, I'd rather prefer:
```suggestion
    Se você tem um artigo, projeto, ferramenta ou qualquer coisa relacionada ao **FastAPI** que ainda não está listada aqui, crie um <a href="https://github.com/tiangolo/fastapi/edit/master/docs/external-links.md" class="external-link" target="_blank">_Pull Request_ para adiciona-lo</a>.
```

Reply:
a minha questão era mais para não cair no problema de "ele(a)" ou  "adicioná-lo(a)". Mas por mim está ok, também, sem problemas.

Reply:
With the arguments above, I would like to keep `adicionando ele`.
@Kludex and @deniscapeto thanks a lot!!

Reply:
Alright. Guys, I'm writing in Portuguese because I've seen Portuguese being used on another code review. I'll use English from now on.

Reply:
No problem. I forgot to say that.